uavcan/equipment/air_data/1027.RawAirData.uavcan

```diff
@@ -0,0 +1,29 @@
+#
+# Raw Air Data.
+#
+
+#
+# Heater State
+# 
+uint8 HEATER_UNAVAILABLE    = 0
+uint8 HEATER_WORKING        = 1
+uint8 HEATER_OVERCURRENT    = 2
+uint8 HEATER_OPENCIRCUIT    = 3
+uint8 heater_state
+
+#
+# Pressure Data
+#
+float32 static_pressure                 # Pascal
+float32 differential_pressure           # Pascal
+
+#
+# Temperature Data
+#
+float16 sensor_temperature              # Kelvin
```

Comment:
As far as I understood yes, as there might be 2 separate sensors. 
However I am still in doubt about this. 
I don't think that anyone ever used precise calibration equipment to check the calibration curves. I expect that the need for such calibration on AP side is coming from wrong reading of outside temperature (without dedicated sensor) and thus the wrong conversion to TAS.
In both cases the issue is present and post calibration on AP side can deal with that to some extent.